clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -381,18 +381,17 @@ public class ConsumerConfig extends AbstractConfig {
     private static final String SECURITY_PROVIDERS_DOC = SecurityConfig.SECURITY_PROVIDERS_DOC;
 
     /**
-     * <code>share.acknowledgement.mode</code> is being evaluated as a new configuration to control the acknowledgement mode
-     * for share consumers. It will be removed or converted to a proper configuration before release.
-     * An alternative being considered is <code>enable.explicit.share.acknowledgement</code> as a boolean configuration.
+     * <code>share.acknowledgement.mode</code> is a config to control the acknowledgement mode
```

Comment:
Just `<code>share.acknowledgement.mode</code>` is sufficient. Because this config is no longer internal, it will appear in the documentation for consumer configs for free.